File: batchbhavna/RestApiExamples/app/src/main/java/com/example/restapiexamples/ApiClient.java
package com.example.restapiexamples;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class ApiClient {
    static String BASE_URL = "https://jsonplaceholder.typicode.com/";

    static Retrofit getRetroft()
    {
        retrofit2.Retrofit retrofitbuilder = new Retrofit.Builder()
                .baseUrl(BASE_URL)
                .addConverterFactory(GsonConverterFactory.create())
                .build();

        return  retrofitbuilder;
    }

}

File: 4pmclass/RestApiExamples/app/src/main/java/com/example/restapiexamples/ApiClient.kt
package com.example.restapiexamples

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object ApiClient {
    val BASE_URL = "https://jsonplaceholder.typicode.com/"
val BASEURLSECon = "https://quizegame537.000webhostapp.com/"
    fun getRetrofit():Retrofit{
      var retrofitbuilder: retrofit2.Retrofit =   Retrofit.Builder()
            .baseUrl(BASEURLSECon )
            .addConverterFactory(GsonConverterFactory.create())
            .build()
        return retrofitbuilder
    }


}